app/serializers/admin_serializers/admin_index_serializer.rb

```diff
@@ -0,0 +1,14 @@
+module AdminSerializers
+  # class AdminIndexSerializer
+  class AdminIndexSerializer < ActiveModel::Serializer
```

Comment:
Please add a relevant name as discussed for naming serializers in this project. Module name should be the controller name and serailzer name should be the action name